diffus/Cargo.toml

```diff
@@ -8,21 +8,18 @@ edition = "2018"
 [dependencies]
 itertools = "0.8"
 
-[dependencies.difference]
-version = "2.0"
-optional = true
-
 [dependencies.indexmap]
 version = "1"
 optional = true
 
-[dependencies.lcs]
-version = "0.2"
+[dependencies.uuid]
+version = "0.7"
 optional = true
 
 [features]
 default = [ "vec-impl", "string-impl" ]
 
-string-impl = [ "difference" ]
-vec-impl = [ "lcs" ]
+string-impl = []
```

Comment:
Remove `string-impl` and `vec-impl` completely? They're part of core funtionality now that they have no dependencies?

Reply:
correct